app/3.0/start/toolbox/deploy.md

```diff
@@ -88,32 +81,40 @@ and create a new project.
     threadsafe: yes
 
     handlers:
-    - url: /bower_components
-      static_dir: build/es5-bundled/bower_components
+    - url: /node_modules
+      static_dir: build/es?-???/node_modules
       secure: always
 
     - url: /images
-      static_dir: build/es5-bundled/images
+      static_dir: build/es?-???/images
       secure: always
 
     - url: /src
-      static_dir: build/es5-bundled/src
+      static_dir: build/es?-???/src
       secure: always
 
     - url: /manifest.json
-      static_files: build/es5-bundled/manifest.json
-      upload: build/es5-bundled/manifest.json
+      static_files: build/es?-???/manifest.json
+      upload: build/es?-???/manifest.json
       secure: always
 
     - url: /service-worker.js
-      static_files: build/es5-bundled/service-worker.js
-      upload: build/es5-bundled/service-worker.js
+      static_files: build/es?-???/service-worker.js
+      upload: build/es?-???/service-worker.js
       secure: always
 
     - url: /.*
-      static_files: build/es5-bundled/index.html
-      upload: build/es5-bundled/index.html
+      static_files: build/es?-???/index.html
+      upload: build/es?-???/index.html
       secure: always
+
+    skip_files:
+    - build/es?-???/
+    - build/es?-???/
+    - images/
+    - node_modules/
+    - src/
+    - test/
```

Comment:
Someone (@keanulee?) brought up another idea for handling this in our meeting today: adding `app.yaml` to the build.

- Create `app.yaml` as above but without the `build/es6-bundled/` prefixes on the paths and without the `skip_files` section.
- Add `"app.yaml"` to the `extraDependencies` array in `polymer.json`.
- Run `polymer build`.
- `cd` into the output directory for the build you want to deploy.
- Run the same `gcloud` commands already listed here.

This way, the `app.yaml` can be a lot simpler and the user can deploy any of the other builds by going to the directory and running `gcloud app deploy` again. The only weird part about how this integrates with the guide is that it has to happen before building but the build section is meant to be deployment agnostic (I think?). I bet it's ok to ask the user to run `polymer build` again during the 'Deploy with App Engine' section though. WDYT?

Reply:
This approach works; if you go with this method be sure to _really_ emphasize the "`cd` into the build output directory" step (users may accidentally run `gcloud deploy` on the project directory, which will upload _everything_ and node_modules/ alone will go over the file limit).